code/datums/supplypacks/munitions.dm

```diff
@@ -46,9 +46,9 @@
 	containername = "Experimental weapons crate"
 	access = access_armory
 
-/datum/supply_packs/munitions/energyweapons
+/datum/supply_packs/munitions/energyweapons 
 	name = "Laser carbine crate"
-	contains = list(/obj/item/weapon/gun/energy/laser = 3)
+	contains = list(/obj/item/weapon/gun/energy/laser = 2) //VOREStation Edit - Made to be consistent with the energy guns crate.
```

Comment:
I'd normally say "just do this in munitions_vr.dm" but I think we should actually send this to Polaris as well.